symplyphysics/laws/electricity/circuits/resistivity_of_parallel_resistors.py

```diff
@@ -0,0 +1,36 @@
+from sympy import Eq, solve
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+)
+from symplyphysics.core.operations.sum_array import SumArray
+from symplyphysics.core.symbols.symbols import tuple_of_symbols
+
+# Description
+## If resistors are connected in parallel, total resistance is the inverse sum of inverse resistance of each resistor.
+## Law: R_parallel = 1/sum(1/R[i]), where
+## R_parallel is total resistance,
+## R[i] is resistance of i-th resistor.
+
+inverse_resistances = Symbol("inverse_resistances", 1 / units.impedance)
+parallel_resistance = Symbol("parallel_resistance", units.impedance)
+law = Eq(parallel_resistance, 1 / SumArray(inverse_resistances), evaluate=False)
+
+
+def print_law() -> str:
+    return print_expression(law)
+
+
+@validate_input(inverse_resistances_=inverse_resistances)
+@validate_output(units.impedance)
+def calculate_parallel_resistance(inverse_resistances_: list[Quantity]) -> Quantity:
```

Comment:
Accept direct resistance here. There is no sense in using inverse resistance as parameter.

Reply:
I understand that, but that would require the law to contain direct resistances too, and I'm not sure what change to make in the law so that it contains a sum of inverted resistances.

Reply:
No, it does not require to change law. Give it a try and ask me for help if you're stuck.

Reply:
I tried to implement a workaround but it seems to require more changes to files (so that mypy doesn't fail) and probably there's a better way to do that